gladier_tools/posix/encrypt.py

```diff
@@ -0,0 +1,52 @@
+from gladier import GladierBaseTool, GladierBaseClient, generate_flow_definition
+from pprint import pprint
+
+
+def encrypt(**data):
+    import os
+    from cryptography.fernet import Fernet
+    from cryptography.hazmat.primitives import hashes
+    from cryptography.hazmat.primitives.kdf.pbkdf2 import PBKDF2HMAC
+    import base64
+
+    password = bytes(data['key'], 'utf-8')
+    salt= b'\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00'
+    # salt = os.urandom(16)
+    kdf = PBKDF2HMAC(algorithm=hashes.SHA256(),length=32,salt=salt,iterations=100000)
+    key = base64.urlsafe_b64encode(kdf.derive(password))
+    fernet = Fernet(key)
```

Comment:
Inline docs here would help a lot. Maybe link to the cryptography docs to explain what some of these things are?

Reply:
Alright, will add!

Reply:
Does the iteration number needs to be passed between endpoints? If yes, it should be a variable in the flow. Otherwise, we may endup throting endpoints with it.